projects/aws/eks-a-snow-admin-ami/Makefile

```diff
@@ -0,0 +1,59 @@
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
```

Comment:
I just changed all the other makefiles to make this a `:=` set to avoid using shell in a "macro".